### PR TITLE
[17.0][ADD] product_pricelist_item_formula_visibility

### PR DESCRIPTION
**Pull request by Anxo82**

### CODE REVIEW

**Comment by Anxo82** [comment]
@dalonsod Review please.
Thanks

File: product_pricelist_item_formula_visibility/views/product_pricelist_item_views.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<odoo>
+    <record id="product_pricelist_item_tree_view_inherit" model="ir.ui.view">
```
Comment (dalonsod) [inline product_pricelist_item_formula_visibility/views/product_pricelist_item_views.xml:2]: Remember also adding these columns in pricelist item list view within pricelist form (view `product.product_pricelist_view`)

File: product_pricelist_item_formula_visibility/views/product_pricelist_item_views.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        <field name="name">product.pricelist.item.tree.inherit</field>
+        <field name="model">product.pricelist.item</field>
+        <field name="inherit_id" ref="product.product_pricelist_item_tree_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='price']" position="before">
+                <field name="compute_price"  string="Computation"/>
+                <field name="base" invisible="compute_price != 'formula'"/>
```
Comment (dalonsod) [inline product_pricelist_item_formula_visibility/views/product_pricelist_item_views.xml]: Make both columns optional (show as default)

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments and please attach screenshots as we can see final location of these new columns

**Comment by Anxo82** [comment]
@dalonsod Review please.
Thanks

**Comment by dalonsod** [comment]
@lmiguens-solvos does it fit requirements?

imagen

The same order in general pricelist item views...

**Comment by lmiguens-solvos** [comment]
> @lmiguens-solvos does it fit requirements?
> 
> imagen
> 
> The same order in general pricelist item views...

In the addon that adds coeficcient, the coefficient had been placed before the price column, so the "new" fields to be displayed would be between minimum quantity and price, and the order is fine: Calculation, Based on and Coefficient.

**Comment by Anxo82** [comment]
@lmiguens-solvos @dalonsod Changed, in original product_pricelist_item_tree_view, min_quantity is after the price, that's why the order is Computatio, Based on and coefficient.
Thanks

**Review by dalonsod — APPROVED** [review]
:+1: 

Please add the following pending stuff:
* Add missing `EOF` empty lines for both views .xml files:

  imagen

* Generate `README` files.
* Squash commits.

**Comment by Anxo82** [comment]
Done, review please.
Thanks

**Review by dalonsod — APPROVED** [review]
Ready to be merged!

**Pull request merged**